Carousel Agenda (old - ignore)

Ldsmiths opened the pull request:
refer to issue #266 

Created a carousel image of an abridged agenda for the Q4 2020 GIS Meeting that links to a PDF of the full agenda

wilsaj commented:
I like it. Head's up, bottom of Data Spotlight looks goofy at md width:
![screen shot 2015-03-25 at 4 48 36 pm](https://cloud.githubusercontent.com/assets/303866/6836289/ca551616-d30e-11e4-9ef7-0732a9914e4c.png)

Commit:
Issue #263

StratMaps Contract page updates

jwhaney committed:
issue #263

products, services, software font awesome icons added. looking at a way to maybe filter by these tags added to each vendor object. in progress...

Commit:
issue #263

added legend for icons, changed services icon

Commit:
issue #266

added carousel image for Q4 2020 meeting agenda that links to pdf of full agenda
